src/browser/app/components/Link.react.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component, PropTypes } from 'react';
+import { Link as ReactRouterLink } from 'react-router';
+
+const styles = {
+  active: {
+    fontWeight: 'bold',
+    textDecoration: 'none',
+  },
+  hover: {
+    textDecoration: 'underline',
+  },
+  link: {
+    color: '#346392',
```

Comment:
At least this belongs to https://github.com/este/este/blob/master/src/common/app/theme.js I believe.